vms/components/avax/base_tx.go

```diff
@@ -65,3 +65,20 @@ func (t *BaseTx) Verify(ctx *snow.Context) error {
 		return nil
 	}
 }
+
+func VerifyMemoFieldLength(memo types.JSONByteSlice, isDurangoActive bool) error {
```

Comment:
I added a different method rather than extending `SyntacticVerify` with config and chainTIme. We can revisit this choice down the line, in case other syntactic checks will depend on the specific fork used